swift/extractor/BUILD.bazel

```diff
@@ -8,7 +8,15 @@ alias(
 
 cc_binary(
     name = "extractor",
-    srcs = ["main.cpp"],
+    srcs = [
+        "Extractor.h",
+        "Extractor.cpp",
+        "Configuration.h",
+        "main.cpp",
+    ],
```

Comment:
I prefer being explicit and have no issues with being stuck with having to update this every time you add/remove a file 😄
I'm happy to switch to globs, though. But before doing that I'd like to see the "final" structure of the files in place. I.e., if we add more folders, then we'd have to change the glob anyways, right?

Reply:
True.

Reply:
personally I prefer globbing, because I think file names are just an implementation detail (and I find keeping up to date the BUILD file just for that to be a bit boring). And having to do the book keeping can lead us not to split big files into smaller ones that might help readability... Regarding separate folders, they should just be separate bazel packages to keep dependencies well separated. So my favourite mode would be to consistently use "flat" globs (`glob(["*.h"])`  for `hdrs`, `glob(["*.cpp"])` for `srcs`), and use separate small bazel packages and `deps`  extensively.

But I'm not adamant about using globbing :)